CMakeLists.txt

```diff
@@ -115,3 +115,4 @@ IF(NOT MR_EMSCRIPTEN)
   add_subdirectory(${PROJECT_SOURCE_DIR}/mrealgorithmspy ./mrealgorithmspy)
 ENDIF() # NOT MR_EMSCRIPTEN
 add_subdirectory(${PROJECT_SOURCE_DIR}/MRTest ./MRTest)
+add_subdirectory(${PROJECT_SOURCE_DIR}/meshconv ./meshconv)
```

Comment:
mb put it under `NOT MR_EMSCRIPTEN` block, i think we don't need it in wasm, it will allow to remove all `IF MR_EMSCRIPTEN` in meshconv/CMakeLists.txt